.github/workflows/task-get-linux-configurations.yml

```diff
@@ -9,7 +9,7 @@ env:
                     'ubuntu:bionic',
                     'rockylinux:8',
                     'rockylinux:9',
-                    'debian:bullseye',
+                    'gcc:12-bullseye',
```

Comment:
Why is this here? Seems unrelated to the PR (at least according to the title).

Reply:
It is to use gcc12, debian images has gcc10, but now I reverted the change.